tuf/ngclient/updater.py

```diff
@@ -218,8 +217,7 @@ def download_target(
             ValueError: Invalid arguments
             DownloadError: Download of the target file failed in some way
             RepositoryError: Downloaded target failed to be verified in some way
-            exceptions.StorageError: Downloaded target could not be written
-                to disk
+            OSError: Cannot open "filepath" for writing.
```

Comment:
strictly speaking there are quite a few reasons for OSError here, not just failing to open file (like not enough space as an example).

Reply:
Updated the message.
Do you think that now is better?

Reply:
I'm not sure there was anything wrong with the original message? Alternatively "Failed to write target to file" could work.

Reply:
I understood you had some reservations about it.
I updated the pr with your suggestion.